server/test/validateRegisterInput.spec.js

```diff
@@ -0,0 +1,32 @@
+const { expect } = require('chai');
+const validateRegisterInput = require('../validation/register');
+
+describe('validateRegisterInput', () => {

```

Comment:
Nice job adding tests! At some point, we will want to mock the MongoDB for testing (can you add an icebox ticket for that)- the reason is we don't want to be adding data to the database while doing tests